### PR TITLE
deploy: finished add farming pools #1

### PR DESCRIPTION
**Pull request by spicysquid168**
## Description
Add farming pools for both BUSD and BNB Vault.
BNB Vault: DOT-BNB, UNI-BNB, LINK-BNB, XVS-BNB, and YFI-BNB
BUSD Vault: VAI-BUSD, USDC-BUSD, DAI-BUSD, and UST-BUSD

## Why?
Adding more pools = distribute liquidity all over the pools, so that yield divider distributed across multiple pools

### CODE REVIEW

**Review by jakeal-papaca — DISMISSED** [review]

File: .testnet.json [hunk 1]
```diff
@@ -71,6 +71,42 @@
...
+        "name": "LINK-WBNB LP",
+        "address": "0x47Ad481dE016436E22a426319A88d0097079F6eE"
+      }, {
+        "pId": 10,
+        "name": "XVS-WBNB LP",
+        "address": "0xd2Adf0F11967E7aDB183C329Ea2b2f45af8bC0d1"
+      }, {
+        "pId": 11,
+        "name": "YFI-WBNB LP",
+        "address": "0xFBFf897F471afdf22b5d8c32f88A529a87a7B81b"
```
Comment (abraham-alpaca) [inline .testnet.json:93]: this is not an LP

File: .testnet.json [hunk 2]
```diff
@@ -71,6 +71,42 @@
...
+      }, {
+        "pId": 12,
+        "name": "VAI-BUSD LP",
+        "address": "0xA61bb010D76eeDae6Bff366cf0cDFD5261069ac1"
```
Comment (abraham-alpaca) [inline .testnet.json:97]: no matching entry

File: .testnet.json [hunk 3]
```diff
@@ -71,6 +71,42 @@
...
+      }, {
+        "pId": 13,
+        "name": "USDC-BUSD LP",
+        "address": "0x0F917dA4AbB16641B2c631913F0a4853D9f274F8"
```
Comment (abraham-alpaca) [inline .testnet.json:101]: no matching entry

File: .testnet.json [hunk 4]
```diff
@@ -71,6 +71,42 @@
...
+      }, {
+        "pId": 14,
+        "name": "DAI-BUSD LP",
+        "address": "0x144Cb4Aff90640B934301FD658cF0587De522eAC"
```
Comment (abraham-alpaca) [inline .testnet.json:105]: no matching entry

File: .testnet.json [hunk 5]
```diff
@@ -71,6 +71,42 @@
...
+      }, {
+        "pId": 15,
+        "name": "UST-BUSD LP",
+        "address": "0x3aa448865beDde0fEb63F2C30D3487A4EC293DA6"
```
Comment (abraham-alpaca) [inline .testnet.json:109]: no matching entry

**Review by abraham-alpaca — APPROVED** [review]

**Review by blackalpacaa — APPROVED** [review]

**Pull request merged**